src/py42/modules/alerts.py

```diff
@@ -29,9 +29,8 @@ def search(self, query):
             :class:`py42.response.Py42Response`: A response containing the alerts that match the given
             query.
         """
-        query = self._add_tenant_id_if_missing(query)
-        uri = self._uri_prefix.format(u"query-alerts")
-        return self._session.post(uri, data=query)
+        alert_client = self._microservice_client_factory.get_alerts_client()
+        return alert_client.search(query)
 
     def get_details(self, alert_ids, tenant_id=None):
```

Comment:
Why do we no longer accept single Id args? This is a breaking change that I think is unnecessary

Reply:
Oh wait it is probably in the client

Reply:
In that case, the doc strings should indicate that. `str or list of str`